.github/workflows/release.yml

```diff
@@ -120,6 +120,7 @@ jobs:
       artifact-name: image-release
       file: Dockerfile
       enable-tests: true
+      multi-arch: true
```

Comment:
Does this not also need the conditional, or should we always be building multi-arch images and only sometimes be pushing them?

Reply:
Right, I'll switch to conditional. We'll need build and push to match.